pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumloadbalancerippools.yaml

```diff
@@ -145,59 +145,80 @@ spec:
               do this validation and update the status to reflect the ability to allocate
               IPs from this pool."
             properties:
-              cidrCounts:
-                additionalProperties:
-                  description: CiliumLoadBalancerIPCounts contains information about
-                    how many IPs have been allocated and are available
+              conditions:
+                description: Current service state
+                items:
+                  description: "Condition contains details for one aspect of the current
```

Comment:
These changes from commit 2 seem to be generated from a later commit (commit 4)?